### DISCUSSION TITLE
Source code of snntorch.Synaptic seems to have a mistake?

### DISCUSSION THREAD

**Discussion by cheese-leopard** [jeshraghian/snntorch#229] [Q&A]
I found that there seems to have a problem with the source code of snntorch's synaptic class. In the following colab link, https://colab.research.google.com/drive/1ntRpP9q-oTeHvkY6TmcfjZ_DUwHzGIR_?usp=sharing
I created a synaptic neuron with init_ Hidden=false and true respectively, but their outputs are different. When init_ hidden=true, synaptic neuron output is the same as leaky neuron output. I think it might be a code error in snntorch/_neurons/Synaptic.py Line 271 

def_ Base_ State_ Function_ Hidden (self, input_):
    base_ fn_ Syn = self.alpha.clamp (0, 1) * self.syn + input_
    base_ fn_ MEM = self.beta.clamp (0, 1) * self.mem + **input_**
    Return base_ FN_ Syn, base_ FN_ MEM

second input_  Should be corrected to base_ fn_ Syn

**Comment by ahenkes1** [comment]
Hi @be-Berserker , could you open an Issue for that?

**Reply by cheese-leopard** [reply]
I have opened one with link: https://github.com/jeshraghian/snntorch/issues/234.
